### PR TITLE
Added an example to test the line-to-line-angle Jacobian and its resi…

### PR DESCRIPTION
**Pull request by juanjqo**
@dqrobotics/developers 

Hi @mmmarinho,

I added the python version of the example recently added in C++, which is used to test the line-to-line-angle Jacobian and its residual.

Best regard, 

Juancho

### CODE REVIEW

File: vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    controller.set_gain(0.5)
+    controller.set_damping(0.1)
+    controller.set_control_objective(ControlObjective.Translation)
+    controller.set_stability_threshold(0.001)
+    #---------------------------------------------------------------
+    vfi_gain = 0.5
+    safe_angle = 15*(pi/180)
+    T = 0.005
+    w = 0.2
+    Alpha = 20*(pi/180)
```
Comment (mmmarinho) [inline vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py]: The uppercase `Alpha` is weird. Any special reason?

**Review by mmmarinho — CHANGES_REQUESTED** [review]

File: vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py (hunk 1 above)
Comment (juanjqo) [reply vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py]: @mmmarinho 
There is no special reason. I guess I was using the classic style definition of a sinusoid in which the amplitude is represented with an uppercase letter.

Source:  [Matlab documentation](https://www.mathworks.com/help/dsp/ref/dsp.sinewave-system-object.html)
Screenshot 2022-11-17 105959

However, to keep the convention of uppercase for matrices and lowercase for scalars, I modified the variable from `Alpha` to `alpha`.

Best regards, 

Juancho

Comment (bvadorno) [reply vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py]: Sorry, @juanjqo, but I have to comment on this one. The classic definition uses the Latin letter `A` for the amplitude, not `Alpha`! 🤣

If this is a local variable that refers to the amplitude, then I suggest using `amplitude`.

Best,
Bruno

Comment (juanjqo) [reply vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py]: @bvadorno you're right! 😂

**Comment by mmmarinho** [comment]
@juanjqo 
Let's change it to `amplitude` so that I can accept this pull request!
(Instead of the current `alpha` I believe).

**Comment by juanjqo** [comment]
@mmmarinho I implemented your suggestion and updated the code based on the current version of the example in C++.
I tested the example again and it is working.

Best regards, 

Juancho

**Pull request merged**